src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,17 +1,61 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { useAppSelector } from '../../app/hooks';
+import { actions as actionsFilter } from '../../features/filter';
 
+enum SelectValue {
+  ALL = 'All',
+  ACTIVE = 'Active',
+  COMPLETED = 'Completed',
```

Comment:
```suggestion
  ALL = 'all',
  ACTIVE = 'active',
  COMPLETED = 'completed',
```
generally that's ok, but here it's needed to pass tests